libs/portals/my-pages/assets/src/screens/VehicleBulkMileageJobDetail/VehicleBulkMileageJobDetail.tsx

```
@@ -115,22 +115,20 @@ const VehicleBulkMileageJobDetail = () => {
+      buttonGroup={[
+        <Button
+          variant="utility"
+          icon={
+            loading || networkStatus === NetworkStatus.refetch
+              ? undefined
+              : 'reload'
+          }
+          loading={loading || networkStatus === NetworkStatus.refetch}
+          onClick={handleRefresh}
+        >
+          {formatMessage(vehicleMessage.refreshJob)}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)